src/Apps/ArtQuiz/ArtQuizResultsLoader.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Box,
+  Flex,
+  Text,
+  Spacer,
+  ArtsyMarkBlackIcon,
+  FullBleed,
+  Spinner,
+} from "@artsy/palette"
+import { SplitLayout } from "Components/SplitLayout"
+import { useNavBarHeight } from "Components/NavBar/useNavBarHeight"
+import { useState, useEffect } from "react"
+import { useTranslation } from "react-i18next"
+
+export const ArtQuizResultsLoader = () => {
+  const { desktop } = useNavBarHeight()
+  const { t } = useTranslation()
+
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false)
+    }, 2000)
```

Comment:
Yea there wasn't any specific guidance on that. I really just tested a few delays to see what would make sense and allow the user to read both without the delay being too long. Do you think it could be longer?

Reply:
yea, I think this is where QA would help surface. I don't think this is blocking.